cranelift/reader/src/parser.rs

```diff
@@ -3678,7 +3678,7 @@ mod tests {
     }
 
     #[test]
-    #[cfg(feature = "riscv")]
+    #[ignore] // FIXME this test is failing for some reason
```

Comment:
Is it intended to mark this test as ignored, or did you plan to look into it? It's not obvious where the error is here; did you get a relevant backtrace or something that would help diagnose what's going on?

Reply:
It just fails to parse the test. I don't know why. This test was already effectively ignored previously as cranelift-reader doesn't have a riscv feature.